app/static/src/app/store/model/mutations.ts

```diff
@@ -32,5 +33,21 @@ export const mutations: MutationTree<ModelState> = {
 
     [ModelMutation.SetRequiredAction](state: ModelState, payload: RequiredModelAction | null) {
         state.requiredAction = payload;
+    },
+
+    [ModelMutation.SetParameterValues](state: ModelState, payload: Dict<number>) {
+        // initialise values
+        state.parameterValues = payload;
+    },
+
+    [ModelMutation.UpdateParameterValues](state: ModelState, payload: Dict<number>) {
```

Comment:
Update with a set of values rather than single value as this will be useful in future ticket when replacing entire parameter sets at once.